fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/pattern/Pattern.java

```diff
@@ -141,6 +141,10 @@ public boolean matchWithoutChild(OptExpression expression) {
             return true;
         }
 
+        if (isPatternMultiJoin() && isMultiJoin(expression.getOp().getOpType())) {
+            return true;
+        }
```

Comment:
Why add this?

Reply:
rule based pattern do not support multi join pattern, so add it here.